src/routes/login/callback/updateUser.ts

```diff
@@ -11,13 +11,11 @@ export async function updateUser(params: {
 	locals: App.Locals;
 	cookies: Cookies;
 }) {
-	console.log("updateUser");
 	const { userData, locals, cookies } = params;
-	userData["preferred_username"] =
```

Comment:
I believe this is required, the login process keep breaking without adding this for me.

Reply:
Should be fixed by https://github.com/huggingface/chat-ui/commit/002a2a0f588a701e0c0f96e2d6c7aef96a8cc512 , can you test?